src/components/PageHeader/DesktopPageHeader.tsx

```diff
@@ -48,6 +49,16 @@ const StyledInnerContainer = styled.div`
   margin: 0 auto;
 `
 
+const StyledClamBalanceContainer = styled.div``
+
+const StyledExchangePopup = styled(ExchangePopup)`
+  position: absolute;
+  display: none;
+  ${StyledClamBalanceContainer}:hover & {
+    display: block;
+  }
+`
+
```

Comment:
We can use `react-tooltip` (we are already using this package elsewhere) directly or use [portal](https://reactjs.org/docs/portals.html) to implement pop-up functionality.

It will cause lots of problems if we use `position: absolute` without portal.